lib/senders/email.js

```diff
@@ -668,6 +742,44 @@ module.exports = function (log) {
     })
   }
 
+  Mailer.prototype.postVerifySecondaryEmail = function (message) {
+    log.trace({ op: 'mailer.postVerifySecondaryEmail', email: message.email, uid: message.uid })
+
+    var templateName = 'postVerifySecondaryEmail'
+    var links = this._generateLinks(this.accountSettingsUrl, message.email, {}, templateName)
```

Comment:
the `Manage Account` link is empty for me in both html and text email:

<img src=https://cloud.githubusercontent.com/assets/128755/24940234/0b4f1dd4-1f10-11e7-8e85-adb387d179af.png width=300 />